sample-code/package.json

```diff
@@ -26,6 +26,8 @@
     "@sap-ai-sdk/ai-api": "workspace:^",
     "@sap-ai-sdk/foundation-models": "workspace:^",
     "@sap-ai-sdk/orchestration": "workspace:^",
+    "@sap-ai-sdk/langchain": "workspace:^",
+    "@langchain/core": "^0.2.30",
```

Comment:
[q] So users would need to install this separately as well to use types?

Reply:
Users can also create these objects on their own or alternatively only pass strings.